swarm/network/kademlia.go

```diff
@@ -439,23 +439,16 @@ func (k *Kademlia) NeighbourhoodDepth() (depth int) {
 	return depthForPot(k.conns, k.MinProxBinSize, k.base)
 }
 
-// depthForPot returns the proximity order that defines the distance of
+// neighbourhoodRadiusForPot returns the proximity order that defines the distance of
 // the nearest neighbour set with cardinality >= MinProxBinSize
 // if there is altogether less than MinProxBinSize peers it returns 0
 // caller must hold the lock
-func depthForPot(p *pot.Pot, minProxBinSize int, pivotAddr []byte) (depth int) {
+func neighbourhoodRadiusForPot(p *pot.Pot, minProxBinSize int, pivotAddr []byte) (depth int) {
```

Comment:
To have a distinction between return values from neighbourhoodRadiusForPot and depthForPot, would it be better to call here a returned value `radius`? And also to call the variable radius in SuggestPeer() function. Just a suggestion.